pkg/services/hivelocity/device/device.go

```diff
@@ -365,6 +379,12 @@ func (s *Service) actionProvisionDevice(ctx context.Context) actionResult {
 	if s.scope.HivelocityCluster.Spec.SSHKey != nil {
 		sshKeyID, err := s.getSSHKeyIDFromSSHKeyName(ctx, s.scope.HivelocityCluster.Spec.SSHKey)
 		if err != nil {
+			if errors.Is(err, errSSHKeyNotFound) {
+				// do not return an error in the reconcile loop as we cannot do anything about this without the intervention
+				// of the user. Only after the SSH key has been uploaded correctly, the provisioning can continue.
+				// This is why we wait for 5m and then reconcile again to see whether the SSH key exists then.
```

Comment:
Where do the 5min come from? I would reconcile faster here. Maybe every 15s.

Reply:
The user has to do something manually here.. That can take a really long time! Would you still say 15 seconds?

Reply:
Changed it to 1 minute now. I guess that's a good compromise ;)